physionet-django/events/views.py

```diff
@@ -210,11 +210,16 @@ def event_detail(request, event_slug):
 
     # Handle Event Registration  End #
```

Comment:
This comment isn't needed (not sure if any of the `Handle Event Datastore` comments are needed tbh)

Reply:
> Thanks Amit, this is an improvement on the original PR and I think it's safe to merge. There's some cleaning to do (and a bigger discussion about how to refactor the code used to control access to content) but we can deal with this later.


Noted. I will keep a list of needed changes and open a small cleanup PR after the datathon.

Also will catchup with you and Alistair to plan the refactoring of access and remaining/other feature of events.